### DISCUSSION TITLE
Frontend 🤪

### DISCUSSION THREAD

**Discussion by LOGESH-B** [gbowne1/codebooker#118] [General]
Hii contributors.....
I have a question 
Why we gone with MUI?Why not Bootstrap or Tailwind?
Is there any specific ,performance related problem!?

**Comment by gbowne1** [comment]
@LOGESH-B 

I made the decision to go with MUI. 

It is fine.  No reason not to and there has been no issue, technical or otherwise, with it in this project.  Yes we could supplement MUI with Tailwind as they can actually work well together but I don't see that as being necessary, unless there is some technical issue with MUI and it provides all we need for this app.

I also do not use Bootstrap outside of native html, css, js projects which have no framework.  It's more suitable for projects such as websites that do not use frameworks.

I would not suggest Bootstrap be used in a  framework project like this one.

**Reply by gbowne1** [reply]
Using TailwindCSS with MUI would require installing Typescript and @mui/core and TailwindCSS. 

https://mui.com/base-ui/guides/working-with-tailwind-css/
